Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
Resize is actually not trigger that often. A quick try with Chrome reveals that trying hard to move it a lot I can get 50 resize events more or less. It would be even less on a less performant computer.

Do you think I we should debounce it? Doesn't the browser cancel previous updates if they get too delayed?